haas/test_common.py

```diff
@@ -388,45 +394,66 @@ def initial_db():
                             username='bob',
                             password='password',
                             type=MockSwitch.api_name)
+        db.session.add(switch)
```

Comment:
@zenhack What I did is that I changed the behavior of the ``test_common.py``, and @knikolla helped me figure out that it actually caused conflicts with ``tests/unit/migrations/falsk.sql``. 

PS: looks like doing auto code format here is really a bad idea.